File: ArraysAndStrings/1.3/Python/urlify.py
def urlify(string, length):
    count = 0
    url = ""
    while count < length:
        if string[count].isspace():
            url += "%20"
        else:
            url += string[count]
        count+=1
    return url

if __name__=="__main__":
    string = input("Introduce input toURLify\n")
    length = int(input("Introduce length\n"))
    url = urlify(string, length)
    print("URL: ", url)

File: ArraysAndStrings/1.3/Golang/urlify.go
package main

import (
	"fmt"
	"bufio"
	"bytes"
	"os"
)

func urlify(originalURL string) string {
	buffer := bytes.Buffer{}
	for _, char := range originalURL {
		if char == ' ' {
			buffer.WriteString("%20")
		} else {
			if char != '\n'{
				buffer.WriteByte(byte(char))
			}
		}
	}
	return buffer.String()
}

func main(){
	reader := bufio.NewReader(os.Stdin)
	fmt.Println("Introduce URL")
	word, _ := reader.ReadString('\n')

	url := urlify(word)

	fmt.Println("URL: "+url)
}